u19_pipeline/lab.py

```diff
@@ -98,6 +98,7 @@ class User(dj.Manual):
     definition = """
     user_id              : varchar(32)                  # username
```

Comment:
Maybe we can have a comment here that instead of saying "# username" says something like "# user_id represents the PNI netid"